### ISSUE TITLE
Make CSS responsive

### ISSUE THREAD

**Issue by geomydas** [majodev/google-webfonts-helper#188]
Thats literally it. Doesn't work on smaller size screens and inaccesible for people who use zoom aswell

**Comment by thomasbritton** [comment]
Apologies if I'm missing something here but wouldn't this be up to the webpage to control and not this service?

**Comment by geomydas** [comment]
I'm not entirely sure aswell. This repo has a webpage [here](https://gwfh.mranftl.com/). It is indeed for the webpage to control.

**Comment by thomasbritton** [comment]
Yeah but the point of this service is to just to provide the font. It's up to the page itself to control the styles of how things are presented or being mobile friendly.

That specific page is just a demo page with instructions on how to use it and just hasn't been made to work on mobile very nicely. So i don't think this an issue with the package it is providing.

**Comment by SAG-Elmar-Hoefinghoff** [comment]
Font declaration doesn't have to be responsive. Which font is used in responsive designs is not controlled by the font declaration. It is done by the website's styles which can than use different font sizes or types from this service depending on viewport or device.
